ui/src/Pages/PeoplePage/SingleFaceGroup/FaceGroupTitle.tsx

```
@@ -148,21 +147,21 @@ const FaceGroupTitle = ({ faceGroup }: FaceGroupTitleProps) => {
-              >
+              onClick={() =>
+                setMergeModalState(MergeFaceGroupsModalState.SelectDestination) // error
+              }
+            >
               {t('people_page.action_label.merge_face', 'Merge face')}
             </Button>
           </li>
           <li>
             <Button onClick={() => setDetachModalOpen(true)}>
-              {t('people_page.action_label.detach_images', 'Detach images')}
+              {t('people_page.action_label.detach_images', 'Detach images')} // error
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)